ocean_lib/common/ddo/test/test_common_ddo.py

```diff
@@ -11,13 +11,16 @@
     ServiceDescriptor,
     ServiceFactory,
 )
+from ocean_lib.common.aquarius.aquarius import Aquarius
 from ocean_lib.common.ddo.ddo import DDO
 from ocean_lib.common.ddo.public_key_base import PublicKeyBase
 from ocean_lib.common.ddo.public_key_rsa import (
     PUBLIC_KEY_TYPE_ETHEREUM_ECDSA,
     PUBLIC_KEY_TYPE_RSA,
 )
 from ocean_lib.common.did import DID
+from ocean_lib.config import Config
```

Comment:
for me, here, there appear to be some imports. Are they used anywhere? Can you please make sure you do not import things which are not used?